router/src/lib.rs

```diff
@@ -19,6 +20,20 @@ pub struct HubModelInfo {
     pub pipeline_tag: Option<String>,
 }
 
+#[derive(Clone, Debug, Serialize, ToSchema)]
+pub struct HealthResponse {}
+
+#[derive(Clone, Debug)]
+pub struct Health {
+    pub client: ShardedClient,
+}
+
+impl Health {
```

Comment:
I guess you don't really need the impl since client is pub

Reply:
I started to do impl as I prefer to pubbing members in general (since modifying attributes on the fly is a rare requirement and it's enabled with pub attributes).

I went the full pub route for homogeneity's sake.